OpenSim/Sandbox/MatlabScripts/createActuatorsFile.m

```diff
@@ -0,0 +1,165 @@
+function createActuatorsFile
+%createActuatorsFile   Build and Print an OpenSim Actuator File from a Model
+%
+%  createActuatorsFile attempts to make a template actuators file that can
+%  be used in Static Optimization, RRA and CMC. This tries to identify the
+%  coordinates  that are connected to ground and place point or torque
+%  actuators on translational or rotational coordinates, respectively. All
+%  other coordiantes will get coordinate actuators. Any locked, prescribed
+%  or constrained coordinates will be ignored.
+
+% ----------------------------------------------------------------------- %
+% The OpenSim API is a toolkit for musculoskeletal modeling and           %
+% simulation. See http://opensim.stanford.edu and the NOTICE file         %
+% for more information. OpenSim is developed at Stanford University       %
+% and supported by the US National Institutes of Health (U54 GM072970,    %
+% R24 HD065690) and by DARPA through the Warrior Web program.             %
+%                                                                         %   
+% Copyright (c) 2005-2012 Stanford University and the Authors             %
+% Author(s): James Dunne                                                  %
+%                                                                         %
+% Licensed under the Apache License, Version 2.0 (the "License");         %
+% you may not use this file except in compliance with the License.        %
+% You may obtain a copy of the License at                                 %
+% http://www.apache.org/licenses/LICENSE-2.0.                             %
+%                                                                         % 
+% Unless required by applicable law or agreed to in writing, software     %
+% distributed under the License is distributed on an "AS IS" BASIS,       %
+% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or         %
+% implied. See the License for the specific language governing            %
+% permissions and limitations under the License.                          %
+% ----------------------------------------------------------------------- %
+
+% Author: James Dunne, Chris Dembia, Tom Uchida,
+% Ajay Seth, Ayman Habib, Shrinidhi K. Lakshmikanth, Jen Hicks.
+
+% Import OpenSim Libraries
+import org.opensim.modeling.*
+
+% open dialog boxes to select the model
+[filename, pathname] = uigetfile('*.osim', 'Select an OpenSim Model File');
+
+% get the model path
+modelFilePath = fullfile(pathname,filename);
+
+% Generate an instance of the model
+model = Model(modelFilePath);
+
+% Get the number of coordinates  and a handle to the coordainte set
+nCoord = model.getCoordinateSet.getSize();
+coordSet = model.getCoordinateSet();
+
+% Evaluate the ground body and get the mass center
+groundBodyName = model.getGround().getName();
+groundJoint = model.getJointSet.get(0);
```

Comment:
I'm not sure it's still guaranteed that the first joint in the JointSet will be connected to Ground.